mandatory/1-fix-functions.js

```diff
@@ -9,10 +9,9 @@
   NOTE:Only make edits inside the function
 
 */
+let isHappy = true;
 
-function mood() {
-  let isHappy = true;
-
+function mood(isHappy) {
```

Comment:
so here you have two `isHappy`. One is a variable declared at line 12 and the other is a parameter of the `mood` function. It's confusing so if you need both you should use different names.